roles/kubernetes-apps/registry/templates/registry-sa.yml.j2

```diff
@@ -4,4 +4,4 @@ metadata:
   name: registry
   namespace: {{ registry_namespace }}
   labels:
-    kubernetes.io/cluster-service: "true"
+
```

Comment:
This is not valid yaml... it should have no label field if there are no labels to apply. Please fix this up.